src/components/NavBar/NavigationBar.tsx

```diff
@@ -0,0 +1,174 @@
+    display: flex;
+    flex-direction: column;
+    justify-content: start;
+    align-items: stretch;
+    padding: 1rem;
+    width: 15rem;
+`;
+
+const Gap = styled.div`
+    width: 1rem;
+`;
+
+const DrawerButtonWrapper = styled.div`
+    padding: 1rem;
+    border-bottom: 1px solid ${(props) => props.theme.secondaryBackgroundColor};
+
+    &:last-child {
+        border-bottom: none;
+    }
+`;
+
+const DrawerButton = styled(Button)`
+    color: ${(props) => props.theme.secondaryBackgroundColor};
+`;
+
+const StickyAppBar = styled(AppBar)`
+    position: sticky;
+`;
+
+const NavElementContainer = styled.div`
+    display: flex;
+    flex-basis: 0;
+    flex-grow: 1;
+    justify-content: center;
+    align-items: center;
+`;
+
+const MobileNavMenuContainer = styled(NavElementContainer)`
+    justify-content: start;
+    @media (min-width: 800px) {
+        display: none;
+    }
+`;
+
+const LogoElementContainer = styled(NavElementContainer)`
+    justify-content: center;
+    height: 100%;
+    object-fit: cover;
+    @media (min-width: 800px) {
+        justify-content: start;
+    }
+`;
+
+const ButtonContainer = styled(NavElementContainer)`
+    display: none;
+    @media (min-width: 800px) {
+        display: flex;
+    }
+`;
+
+const SignOutButtonContainer = styled(NavElementContainer)`
+    justify-content: end;
+`;
+
+const ResponsiveAppBar: React.FC = () => {
+    const [drawer, setDrawer] = React.useState(false);
+
+    const openDrawer = (): void => {
+        setDrawer(true);
+    };
+
+    const closeDrawer = (): void => {
+        setDrawer(false);
+    };
+
+    const pages = [
+        ["Clients", "/clients"],
+        ["Lists", "/lists"],
+        ["Calendar", "/calendar"],
+    ];
+
+    return (
+        <>
+            <SwipeableDrawer open={drawer} onClose={closeDrawer} onOpen={openDrawer}>
+                <DrawerInner>
+                    {pages.map(([page, link]) => (
+                        <DrawerButtonWrapper key={page}>
+                            <UnstyledLink href={link} onClick={closeDrawer} prefetch={false}>
+                                <DrawerButton variant="text">{page}</DrawerButton>
+                            </UnstyledLink>
+                        </DrawerButtonWrapper>
+                    ))}
+                </DrawerInner>
+            </SwipeableDrawer>
+            <StickyAppBar>
+                <AppBarInner>
+                    <MobileNavMenuContainer>
+                        <IconButton aria-label="Mobile Menu Button" onClick={openDrawer}>
+                            <MenuIcon />
+                        </IconButton>
+                    </MobileNavMenuContainer>
+                    <LogoElementContainer>
+                        <UnstyledLink href="/" prefetch={false}>
+                            <Logo alt="Vauxhall Foodbank Logo" src="/logo.webp" />
+                        </UnstyledLink>
+                    </LogoElementContainer>
+                    <ButtonContainer>
```

Comment:
Should this be `DesktopButtonContainer`?

Reply:
Yes, has been changed